e3tran/BitFit
# Trophy

Merge Trophy branch which contains frontend and backend code for the trophy case.

To Test:
Make sure to have the app frontend running, and the backend.

0. Login with test@gmail.com, test123
1. Press the trophy case button.
2. Loading screen will show up briefly.
3. 1 Trophy should earned and show up as a trophy. The rest should not be earned and show up as a lock.
4. Click on the earned trophy, and a popup will display with the trophy name and saying congrats and its description.
5. Click on a locked trophy, and a pop up will display saying the trophy name and your progress towards that trophy.
6. Navigate back to the Progress page successfully.
